internal/services/vpcgw/network.go

```diff
@@ -85,6 +93,7 @@ func ResourceNetwork() *schema.Resource {
 				Computed:      true,
 				ValidateFunc:  validation.IsCIDR,
 				ConflictsWith: []string{"dhcp_id", "ipam_config"},
+				Deprecated:    "Please use ipam_config",
```

Comment:
Add a link to the migration guide with a more explicit guide that IPAM is the way forward.